### ISSUE TITLE
Only show numbers if red?

### ISSUE THREAD

**Issue by Docteh** [shedaniel/LightOverlay-Fabric#62]
I noticed that if i want to use optifine shaders with the LightOverlay, I need to use the numbers mode, but it shows numbers for every block, including on top of torches.  (This is mentioned on #5 and the [curseforge page](https://www.curseforge.com/minecraft/mc-mods/light-overlay))

Would it be possible to only draw numbers when they would be 7  or lower?

Whenever I'm digging or mining, the numbers blink green boxes, and if I could configure it to only show on the low light blocks, this wouldn't be a big problem. 
![image](https://user-images.githubusercontent.com/190571/93392957-801c3780-f826-11ea-902b-ee08f22c9df9.png)

I looked at the code for a potential easy spot to add a cheeky `if (lightLevel >=7) { ... }` but I don't understand minecraft mods 😞

**Comment by zeehtaa** [comment]
BUMP!
I'd love this feature too, would be great to have a config value to enable a mode where it only shows
`light level <= light level config value` and hides all the other (non-relavant) information.

Also.. thanks for a super userful mod 🍰 😃

**Comment by PieterVMB** [comment]
I also would love to see this option.

**Comment by lexi-the-cute** [comment]
yep! while this can be kinda done with the crosses and turning off the secondary light level, but it can't be done with the numbers setup
